documentation/modules/configuring/con-config-high-volume-messages.adoc

```diff
@@ -0,0 +1,238 @@
+// Module included in the following assemblies:
+//
+// assembly-config-high-volumes.adoc
+
+[id='con-high-volume-config-properties-{context}']
+= Handling high volumes of messages
+
+[role="_abstract"]
+If your Strimzi deployment needs to handle a high volume of messages, you can use configuration options to optimize for throughput and latency. 
+
+Producer and consumer configuration can help control the size and frequency of requests to Kafka brokers. 
+For more information on the configuration options, see the following:
+
+* xref:con-producer-config-properties-throughput-{context}[] 
+* xref:con-consumer-config-properties-throughput-{context}[]  
+
+You can also use the same configuration options with the producers and consumers used by Kafka Connect runtime and MirrorMaker 2.0 source connectors and sink connectors.
```

Comment:
What about:
```
You can also use the same configuration options with the producers and consumers used by the Kafka Connect runtime, source connectors (including and MirrorMaker) and sink connectors.
```

Reply:
Changed to _used by the Kafka Connect runtime source connectors (including MirrorMaker 2.0) and sink connectors._